### DISCUSSION TITLE
multiple file paths for data in Docker

### DISCUSSION THREAD

**Discussion by cyberlauncher** [stashapp/stash#3213] [Q&A] [answered]
Stash running under Docker here.
Anyone know how I can have multiple paths to different collections for the data path in Docker?
For example if I have one collection on /alpha/ and another on /beta/

Could I structure the docker-compose.yml like -
 **/dev/disk/by-label/alpha /dev/disk/by-label/beta/ _(etc)_:/data**

**Comment by WithoutPants** [comment, marked as answer]
I would try the following changes to the `docker-compose.yml` file:
- remove the `STASH_STASH` environment variable
- add two `volumes` entries:
  - `/dev/disk/by-label/alpha:/data`
  - `/dev/disk/by-label/beta:/data2`
- run the docker image, then within the settings, add `/data` and `/data2` to your stash libraries.

**Reply by cyberlauncher** [reply]
Perfect. Thank you!
